packages/realm/src/OrderedCollection.ts

```diff
@@ -364,14 +373,37 @@ export abstract class OrderedCollection<T = unknown, EntryType extends [unknown,
     throw new Error("Method not implemented.");
   }
 
+  /**
+   * Checks if this collection has not been deleted and is part of a valid Realm.
+   * @returns `true` if the collection can be safely accessed, `false` if not.
+   * @since 0.14.0
+   */
   isValid(): boolean {
     throw new Error(`Calling isValid on a ${this.constructor.name} is not support`);
   }
 
+  /**
+   * Checks if this collection is empty.
+   * @returns `true` if the collection is empty, `false` if not.
+   * @since 2.7.0
+   */
   isEmpty(): boolean {
     return this.results.size() === 0;
   }
 
+  /**
+   * Returns the minimum value of the values in the collection or of the
+   * given property among all the objects in the collection, or `undefined`
+   * if the collection is empty.
+   *
+   * Only supported for int, float, double and date properties. `null` values
+   * are ignored entirely by this method and will not be returned.
+   *
+   * @param property For a collection of objects, the property to take the minimum of.
+   * @throws {Error} If no property with the name exists or if property is not numeric/date.
```

Comment:
Not a blocker, but this could be more specific.

```suggestion
   * @throws {TypeAssertionError} If no property with the name exists or if property is not numeric/date.
```